singularity-eos/CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ set(EOS_HEADERS
     eos/eos_vinet.hpp
     eos/eos_builder.hpp
     eos/eos_jwl.hpp
+    eos/eos_helmholtz.hpp
```

Comment:
Not an issue to resolve here, but I'm a little worried that we include headers that are not used with specific build parameters (e.g. `_USE_HELMHOLTZ` or `_USE_EOSPAC`). We should go through these in the future and guard their inclusion.